src/components/inputs/Inputs.jsx

```diff
@@ -2232,14 +2233,20 @@ const ReferenceTable = observer(({
                           ConfirmDelete({
                             itemName: itemName,
                             onConfirm: () => {
-                              store.RemoveField({
+                              store.ApplyTransaction( {
```

Comment:
this ApplyTransaction call needs all of the info passed to RemoveField, since it will replace the remove action (page, path, cat/subcat, label). You'll know you did it right if it shows up correctly in the changelist when you hit save